p2p/libp2p/libp2pConnectionMonitor.go

```diff
@@ -7,11 +7,15 @@ import (
 
 	"github.com/ElrondNetwork/elrond-go/core/check"
 	"github.com/ElrondNetwork/elrond-go/p2p"
-	"github.com/ElrondNetwork/elrond-go/p2p/libp2p/networksharding"
+	ns "github.com/ElrondNetwork/elrond-go/p2p/libp2p/networksharding"
 	"github.com/libp2p/go-libp2p-core/network"
 	"github.com/multiformats/go-multiaddr"
 )
 
+const (
+	watchdogTimeout = 5 * time.Minute
+)
+
 // DurationBetweenReconnectAttempts is used as to not call reconnecter.ReconnectToNetwork() to often
```

Comment:
typos:
to often -> too often
succeed -> succeeded/succeeds

Reply:
done